source/auth/auth.rst

```diff
@@ -613,13 +613,7 @@ This same conversation over mongodb's sasl implementation would appear as follow
 | C: :javascript:`{saslStart: 1, mechanism: "SCRAM-SHA-1", payload: BinData(0, "biwsbj11c2VyLHI9ZnlrbytkMmxiYkZnT05Sdjlxa3hkYXdM")}`
```

Comment:
In order for it to work this way the driver has to send `options: { skipEmptyExchange: true }`

Reply:
Earlier versions of the server will ignore this, and the longer conversation will be used instead, and drivers have to be able to handle both.

Reply:
See https://jira.mongodb.org/browse/SERVER-44857 for details.

Reply:
☝️ What @jyemin said.  I think if you add `options: { skipEmptyExchange: true }` here (and not below), then this diff should be fine.

We probably want to leave it out of the lower place since that spot is illustrating what a "long" exchange looks like.

Reply:
I have added the `options` document.

Reply:
I would leave it in the "long" exchange because the driver is always going to add it